src/lib/tabs/Tabs.svelte

```diff
@@ -120,36 +123,35 @@
 	});
 </script>
 
-<div
-	data-border={border}
-	class="flex place-items-center data-[border=true]:border-b border-gray-100 overflow-hidden
-dark:border-gray-900 relative group/tabs"
->
+
+<div data-border={border}
+	 class={cn('flex place-items-center overflow-hidden data-[border=true]:border-b border-gray-100 relative group/tabs ', className)}>
+
```

Comment:
When you copied this is clipped out some of the class names and the correct border color is not being applied in dark mode.
Should change to:
```svelte
class={cn('flex place-items-center data-[border=true]:border-b border-gray-100 overflow-hidden
dark:border-gray-900 relative group/tabs', className)}>
```

Reply:
please check